capa/features/extractors/binexport2/__init__.py

```diff
@@ -0,0 +1,405 @@
+# Copyright (C) 2023 Mandiant, Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at: [package root]/LICENSE.txt
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+#  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+"""
+Proto files generated via protobuf v24.4:
+
+    protoc --python_out=. --mypy_out=. binexport2.proto
```

Comment:
should we vendor the `binexport2.proto` file that we used to generate the bindings, so that its easy to compare to future versions? otherwise, we might have a hard time figuring out if our bindings are up to date.

Reply:
I agree and I'm open to suggestions on the best way to do this.

Reply:
Can we link/document the respective commit for https://github.com/google/binexport/blob/main/binexport2.proto?

Reply:
added in #2353